Sprint2/5-Cost-analysis.md

```diff
@@ -0,0 +1,1647 @@
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Descripción</strong>
+   </td>
+   <td colspan="2" ><strong>Coste por hora</strong>
+   </td>
+   <td colspan="2" ><strong>Coste por mes (10h/semana)</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td><strong>Bruto</strong>
+   </td>
+   <td><strong>Neto</strong>
+   </td>
+   <td><strong>Bruto</strong>
+   </td>
+   <td><strong>Neto</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Costes de salario
+   </td>
+   <td>10,26€
+   </td>
+   <td>7,18€
+   </td>
+   <td>410,40€
+   </td>
+   <td>287,28€
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Total (37 semanas)</strong>
+   </td>
+   <td colspan="4" ><strong>3693,6€</strong>
+   </td>
+  </tr>
+</table>
+
+
+<br/>
+
+## 2.5 Amortización en producción
+
+
+<table>
+  <tr>
+   <td colspan="2" ><strong>Amortización en producción</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Periodo producción
+   </td>
+   <td> 37 semanas
+   </td>
+  </tr>
+  <tr>
+   <td>Amortización semanal
+   </td>
+   <td>200€
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Total</strong>
+   </td>
+   <td><strong>7.400€</strong>
+   </td>
+  </tr>
+</table>
+
+
+Teniendo esto en cuenta, durante 37 semanas, que junto con las 15 semanas de desarrollo completamos el 1º año de vida del producto, amortizamos **7.400€.**
+
+<br/>
+
+## 2.6 Desglose total de los costes de producción
+
+Como aclaración, el coste de periodo de mantenimiento es el salario **BRUTO**, no neto.
+
+
+<table>
+  <tr>
+   <td><strong>Descripción</strong>
+   </td>
+   <td><strong>Duración</strong>
+<p>
+<strong>(Semanas)</strong>
+   </td>
+   <td><strong>Coste</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Periodo de Mantenimiento
+<p>
+(Personal)
+   </td>
+   <td>37
+   </td>
+   <td>26.177,50€
+   </td>
+  </tr>
+  <tr>
+   <td>Costes derivados del mantenimiento (internet y luz)
+   </td>
+   <td>37
+   </td>
+   <td>3.330€
+   </td>
+  </tr>
+  <tr>
+   <td>Costes de licencias
+   </td>
+   <td>37
+   </td>
+   <td>4.114,28€
+   </td>
+  </tr>
+  <tr>
+   <td>Costes de márketing
+   </td>
+   <td>37
+   </td>
+   <td>3693,6€
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Total</strong>
+   </td>
+   <td>37
+   </td>
+   <td><strong>37.315,38€</strong>
+   </td>
+  </tr>
+</table>
+
+
+<br/>
+
+# 3. Coste posterior de la ONG
+
+Podemos hacer referencia a dos costes seguros que va a tener la ONG posteriores a nuestra finalización del proyecto, estos son el coste del dominio y el coste del despliegue.
+
+El coste de dominio lo sacamos de la página de IONOS, compramos el dominio .es y en el caso del coste de despliegue utilizando el despliegue F2 de app engine, que son 0.10 euros la hora por el coste del despliegue de la aplicación.
+
+
+<table>
+  <tr>
+   <td colspan="2" ><strong>Costes posteriores de la ONG</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Coste del dominio
+   </td>
+   <td><strong>50€</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Coste del despliegue
+   </td>
+   <td><strong>876€</strong>
```

Comment:
I understand your comment, but I do talk about the deployment cost in this section. Perhaps you would like me to also address this in another section. What section would that be?

Reply:
True. You are totally correct. But that cost, is monthly or annual?